src/main/java/forestry/apiculture/worldgen/HiveDescription.java

```diff
@@ -146,6 +146,10 @@ public float getGenChance() {
 
 	@Override
 	public void postGen(World world, Random rand, BlockPos pos) {
+		if(isGoodBiome(world.getBiome(pos)))
+		{
+			Blocks.FIRE.setFireInfo(world.getBlockState(pos).getBlock(), 5, 5);

```

Comment:
This does not work like you think. `setFireInfo` sets the fire info for all forestry hives because all forestry hives have the same block instance. You have to implement the methods`getFlammability`, `getFireSpreadSpeed` and `isFlammable` in the class of the hive block.

Reply:
Ah OK. Makes sense.